src/architecture/transactions.md

```diff
@@ -220,30 +216,49 @@ via [an appropriate transaction endpoint](services.md#transactions).
 
 !!! note
     As transactions use universally verified cryptography (digital signatures)
-    for authentication, a transaction theoretically can be submitted to the network
-    by anyone aware of the transaction. There is no intrinsic upper bound on
-    the transaction lifetime, either.
+    for authentication, a transaction theoretically can be submitted to the
+    network by anyone aware of the transaction. There is no intrinsic upper
+    bound on the transaction lifetime, either.
 
 !!! tip
     From the point of view of a light client, transaction execution is
     asynchronous; full nodes do not return an execution status synchronously
-    in a response to a client’s request. To determine transaction status,
-    you may poll the transaction status using [read requests](services.md#read-requests)
-    defined in the corresponding service or the blockchain explorer.
-    If a transaction is valid (i.e., its `verify` returns `true`), it’s expected
-    to be committed in a matter of seconds.
+    in response to the request of the client. To determine the transaction status,
+    you may poll the transaction status using
+    [read requests](services.md#read-requests) defined in the corresponding
+    service or the blockchain explorer. If a transaction is valid, it is
+    expected to be committed in a matter of seconds.
 
 ### 3. Verification
 
 After a transaction is received by a full node, it is looked up
 among committed transactions, using the transaction hash as the unique
-identifier. If a transaction has been committed previously, it is
+identifier. If the transaction has been committed previously, it is
 discarded, and the following steps are skipped.
 
 The transaction implementation is then looked up
 using the `(service_id, message_id)` type identifier.
-The `verify` method of the implementation is invoked to check the internal
-consistency of the transaction.
+
+If the transaction is absent in the blockchain, the `raw_from_buffer` method
```

Comment:
What do you mean by "transaction is absent in the blockchain"?

Reply:
That a transaction has not been included into the blockchain before.
Are perhaps some changes required here?